.helm/assure-hmrc-data/templates/_envs.tpl

```diff
@@ -121,4 +121,19 @@ env:
         name: assure-hmrc-data-application-output
         key: mock_azure_password
   {{ end }}
+  - name: REDIS_URL
+    valueFrom:
+      secretKeyRef:
+        name: elasticache
+        key: redis_url
+  - name: SIDEKIQ_USERNAME
+    valueFrom:
+      secretKeyRef:
+        name: assure-hmrc-data-application-output
+        key: sidekiq_username
+  - name: SIDEKIQ_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: assure-hmrc-data-application-output
+        key: sidekiq_password
```

Comment:
https://github.com/ministryofjustice/laa-assure-hmrc-data/pull/161/files#r1183396820

Reply:
these secrets will need adding to staging and production too before merge. currently can't see them in there